aiida/__init__.py

```diff
@@ -42,18 +42,6 @@
 )
 __paper_short__ = 'G. Pizzi et al., Comp. Mat. Sci 111, 218 (2016).'
 
-# Configure the default logging
-configure_logging()
```

Comment:
By removing this the logging will now not be configured until a profile is loaded. This is probably OK in most cases, but since we allow importing AiiDA and running parts of the code without a profile having been loaded, it might happen that logging is needed before the profile is loaded. This is the reason I already put this statement here and made the profile dependent part optional. I think we should keep it

Reply:
At first I left it in but it turns out that `configure_logging()` needs the profile (currently).
Of course one could change the implementation, but I wonder (I really don't know): what is the logging used for, if no profile is loaded?